projects/plugins/automattic-for-agencies-client/src/js/components/admin-page/index.jsx

```diff
@@ -20,69 +20,20 @@ const Admin = () => {
 	const { isUserConnected, isRegistered } = connectionStatus;
 	const showConnectionCard = ! isRegistered || ! isUserConnected;
 	return (
-		<AdminPage
-			moduleName={ __( 'Automattic For Agencies Client', 'automattic-for-agencies-client' ) }
-		>
-			<AdminSectionHero>
-				{ showConnectionCard ? (
+		<ThemeProvider targetDom={ document.body }>
```

Comment:
Added `<ThemeProvider>` to ensure CSS variables are available for Jetpack components. Using the `targetDom` prop here to ensure that popups and modals can access the variables.